gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -142,6 +142,7 @@ management:
 #    keystore:                    # path to KeyStore (when sslEnabled is true, default null)
 #    keystorePassword:            # KeyStore password (when sslEnabled is true, default null)
 #    keyPassword:                 # password for recovering keys in the KeyStore (when sslEnabled is true, default null)
+#    tlsProtocol:                 # protocol to use when connecting to the mongodb instance (when sslEnabled is true, default 'TLSv1.2')
```

Comment:
What about the gravitee.yml for the GW ?

Reply:
Ah ha! Good catch, fixed.